services/rds/rds.go

```diff
@@ -502,6 +502,12 @@ func (svc *Service) updateRDSExporterConfig(tx *reform.TX, service *models.RDSSe
 	}
 	for _, n := range nodes {
 		node := n.(*models.RDSNode)
+
+		var service models.RDSSeree
```

Comment:
`RDSSeree`?

Reply:
@percona-csalguero please build it locally